webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
- Should this be integrated into the existing build step?
- Since we create many different build variants anyways, is `hooks/build/wp-core` the best location, or should we have a `hooks/build-core` or similar?
- Instead of explicitly setting `window` globals in `src/wp-hooks.js`, you might consider [Webpack's `library` and `libraryTarget` options](https://webpack.js.org/configuration/output/#output-library)
- The gap has narrowed in recent Webpack releases, but I've tended toward Rollup as a bundler for library code specifically ([related article](https://medium.com/webpack/webpack-and-rollup-the-same-but-different-a41ad427058c)). Not really pushing it, but raising for discussion nonetheless.